### ISSUE TITLE
Cannot read property 'match' of undefined

### ISSUE THREAD

**Issue by asakusuma** [ember-template-lint/ember-template-lint#1066]
I built a standalone plugin module, which uses the latest `ember-template-lint` (1.x), as a devDependency, for testing the rules defined in the plugin module.

When I yarn link the plugin module into a project which uses the latest non-beta version of `ember-cli-template-lint` (0.7.x), I get this error on `ember build`:

```
dummy/templates/index.hbs
  -:-  error  Cannot read property 'match' of undefined  undefined
```

I traced the error to [this line in `base.js`](https://github.com/ember-template-lint/ember-template-lint/blob/master/lib/rules/base.js#L61).

If I run `./node_modules/.bin/ember-template-lint dummy/templates/index.hbs`, I get no error.

If I change the projects version of `ember-cli-template-lint` to the latest beta (`1.0.0-beta3`), `ember build` no longer has any errors.

I suspect this is caused by the project and plugin module using different major versions of `ember-template-lint`, which may be caused by yarn link.

**Comment by dcyriller** [comment]
It might be related to https://github.com/ember-template-lint/ember-template-lint/issues/1065

**Comment by dcyriller** [comment]
`ember-cli-template-lint` is bringing an outdated version of `ember-template-lint`. This issue looks fixed in `ember-template-lint`'s latest versions. The solution is indeed to upgrade `ember-cli-template-lint`. I have released a v1.0.0.

I'm closing this issue. If you need, don't hesitate to re-open.

**Comment by asakusuma** [comment]
Thanks @dcyriller !
